runtime/doc/lua.txt

```diff
@@ -3375,6 +3375,15 @@ vim.json.encode({obj}, {opts})                             *vim.json.encode()*
       • {opts}  (`table<string,any>?`) Options table with keys:
                 • escape_slash: (boolean) (default false) Escape slash
                   characters "/" in string values.
+                • pretty: (table) Table with keys:
```

Comment:
What kind of error do we show if the provided formatting options result in invalid JSON?